docs/running.rst

```diff
@@ -65,6 +65,39 @@ You may also want to enable human readable logs in the shared config file:
     ; Uncomment to provide stdout logging in a more human readable format.
     #human_logs
 
+Configuring SSL/TLS
+-------------------
+
+The Push protocol mandates use of SSL/TLS connections to ensure the integrity
+of push subscription data. This rule can be ignored for testing purposes
+only. This requires a valid, trusted TLS certificate (for example, from `Let's
+Encrypt`_).
+
+The ``ssl_key`` and ``ssl_cert`` options configure a TLS certificate for both
+the endpoint's HTTPS and connection node's WebSocket Secure (WSS) connections:
+
+.. code-block:: ini
+
+    ; TLS private key and certificate file paths, used for the main
+    ; connection and endpoint listeners. If omitted, TLS will not be
+    ; used.
+    ssl_key = /etc/letsencrypt/live/EXAMPLE.COM/privkey.pem
+    ssl_cert = /etc/letsencrypt/live/EXAMPLE.COM/cert.pem
+    ; Path to file containing any additional parameters for the Ephemeral
+    ; Diffie-Hellman
+    #ssl_dh_param =
+
+The connection between the endpoint and connection nodes should not be exposed
```

Comment:
"exposed" or "accessible"? Really, it should be locked to only other push server machines, but that might be more complicated than necessary.